css/styles.css

```diff
@@ -281,6 +281,10 @@ a {
     overflow: hidden;
 }
 
+.img-40 {
```

Comment:
Thanks for the PR!

This class name is not descriptive of its behavior, though I think we have some other technical debt causing this issue.

Reply:
I didn't want to interfere  with other possible JS or other modules relying on this selector.
My suggested change is a simple adjustment to the image size to make sure it doesn't overflow on small screen resolutions.